src/open_inwoner/utils/ckeditor.py

```diff
@@ -28,15 +30,77 @@ def get_rendered_content(content):
         for element in soup.find_all(tag):
             element.attrs["class"] = class_name
             if element.name == "a" and element.attrs.get("href", "").startswith("http"):
-                icon = soup.new_tag("span")
-                icon.attrs.update(
-                    {
-                        "aria-hidden": "true",
-                        "aria-label": "Opens in new window",
-                        "class": "material-icons",
-                    }
-                )
-                icon.append("open_in_new")
-                element.append(icon)
+                element.attrs["target"] = "_blank"
+
+    return soup
+
+
+def get_product_rendered_content(product):
+    """
+    Takes product's content as an input and returns the rendered one.
+    """
+    md = markdown.Markdown(extensions=["tables"])
+    html = md.convert(product.content)
+    soup = BeautifulSoup(html, "html.parser")
+    class_adders = [
+        ("h1", "h1"),
+        ("h2", "h2"),
+        ("h3", "h3"),
+        ("h4", "h4"),
+        ("h5", "h5"),
+        ("h6", "h6"),
+        ("img", "image"),
+        ("li", "li"),
+        ("p", "p"),
+        ("a", "link link--secondary"),
+        ("table", "table table--content"),
+        ("th", "table__header"),
+        ("td", "table__item"),
+    ]
```

Comment:
If this is the same data for each function then it is better to move it out of the function to a shared variable so we don't forget to update both if we need changes in the future.